InkscapeExtension/ExtendedHatchFill.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+# coding=utf-8
+
+#
+"""
+This extension changes the fill of all selected elements to blue.
```

Comment:
It is a good idea to write complete documentation. But because "fill with blue" is a temporary functionality and we know in advance that it will change soon, I think it should probably documented in the pull request description.